### ISSUE TITLE
CRC 

### ISSUE THREAD

**Issue by dimitrirostavo** [digitalbazaar/forge#280]
Is there a way to calculate the CRC of buffer that is generated from forge.random.getBytesSync ?

**Comment by dlongley** [comment]
Yes, but, unfortunately, there's no utility function in forge for it at the moment. It's not terribly complex to write. PRs are welcome!
